### DISCUSSION TITLE
Problem with configuring a new robot

### DISCUSSION THREAD

**Discussion by MonoJRz** [NVlabs/curobo#420] [Software Q&A]
I’m currently following a robot configuration tutorial using the xArm6, and I’ve reached a self-collision configuration. However, when I attempt to run it, I encounter the following error: 
![Screenshot from 2024-10-24 18-53-01](https://github.com/user-attachments/assets/61e5e41b-a81e-466f-b481-53db38377e90)
Has anyone else experienced this issue or does anyone have advice on how to resolve it? Your assistance would be greatly appreciated.

 Thank you in advance

**Comment by balakumar-s** [comment]
Can you share the full yml file?

**Reply by MonoJRz** [reply]
[xarm6.zip](https://github.com/user-attachments/files/17541579/xarm6.zip)The former is  the yaml file only and the later is all of it. Please let me know if you want any other files.
[robotic.zip](https://github.com/user-attachments/files/17541599/robotic.zip)
